massa-bootstrap/src/server.rs

```diff
@@ -432,7 +432,10 @@ async fn manage_bootstrap(
     loop {
         match tokio::time::timeout(bootstrap_settings.read_timeout.into(), server.next()).await {
             Err(_) => return Ok(()),
-            Ok(Err(e)) => return Err(e),
+            Ok(Err(e)) => match e {
+                BootstrapError::IoError(_) => return Ok(()),
```

Comment:
why is IoError OK ?

Reply:
The client doesn't send a message when the bootstrap is finish he just end the socket and the server doesn't manage any client state so the server can't know that a bootstrap is well performed, he can know that steps independently are performed. It's our design choice. So the only way the server can assume the client bootstrapped it's when he disconnect. Of course, the client can disconnect when he just have connection problem but I don't think it will yield the same error and if it the case then the server has no way to know if it's in case of an end or disconnection.

Reply:
Do you think we could imagine adding a client message confirming successful bootstrap before disconnecting, just like we do for errors ?

Reply:
Personally I prefer with an end message, yes. But I understood that we want to have a server that just answer to ask of client and when it don't ask anything then well we disconnect him.

Reply:
Ok I do it.

Reply:
Done @damip